F/flex/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/westes/flex/releases/download/v$(version)/flex-$(version).tar.gz",
+                  "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gettext-dev
+cd ${WORKSPACE}/srcdir/flex-*
+if [[ "${target}" == *-mingw* ]]; then
+	cp ${prefix}/include/pcreposix.h ${prefix}/include/regex.h
+	export LDFLAGS="-lpcreposix-0 -L${prefix}/bin"
```

Comment:
```suggestion
	cp ${includedir}/pcreposix.h ${includedir}/regex.h
	export LDFLAGS="-lpcreposix-0 -L${libdir}"
```

Reply:
I didn't look into the configure log, but my educated guess is that you need to add `${includedir}` to the search path for the header files